packages/jellyfish-api-core/src/category/mining.ts

```diff
@@ -37,6 +37,16 @@ export class Mining {
   async getMiningInfo (): Promise<MiningInfo> {
     return await this.client.call('getmininginfo', [], 'number')
   }
+
+  /**
+   *
+   * @param {number} confirmationTarget in blocks (1 - 1008)
+   * @param {EstimateMode} [estimateMode='CONSERVATIVE'] estimateMode of fees.
+   * @returns {Promise<SmartFeeEstimation>}
+   */
+  async estimateSmartFee (confirmationTarget: number, estimateMode: wallet.Mode = wallet.Mode.CONSERVATIVE): Promise<SmartFeeEstimation> {
```

Comment:
don't import `wallet` even though it's applicable. This is to prevent code smell where everything starts to depend on each other, making it hard to migrate around. For abstraction, yes, but for implementation no. `wallet` is an implementation.

Reply:
ok will update it